lux/executor/PandasExecutor.py

```diff
@@ -317,14 +318,20 @@ def compute_dataset_metadata(self, ldf:LuxDataFrame):
         self.compute_data_model(ldf)
 
     def compute_data_type(self, ldf:LuxDataFrame):
+        from pandas.api.types import is_datetime64_any_dtype as is_datetime
+        temporal_var_list = ["month", "year","day","date","time"]
```

Comment:
For the temporal variable name check, should we include other potential temporal variables to the list? Eg: hour, minute, second, etc.

Reply:
This was code that was there already, so I left it as is. I think it's reasonable to add the others as well. At the same time, I'm not sure that reading column names to detect temporal is good practice to begin with. For example, one of my datasets had "review_month" as a column. That was just recognized as an integer column. This will also fail on columns that are titled "month", but contain string values (ex. January, February, etc.). Thoughts?

@dorisjlee thoughts?

Reply:
Can we actually remove this code altogether?
```
 elif str(attr).lower() in temporal_var_list:
                ldf.data_type_lookup[attr] = "temporal"
```
and rely on `is_datetime_string` to do the automatic detection? 
Like you mentioned, the string checking technique leads to a lot of false positives.

Reply:
I had actually removed this prior to making the pull request, but it broke the case where you have an int column named 'year'. That may actually be the desired behavior, but we'll just need to adjust test cases accordingly.